R/zzz.R

```diff
@@ -12,28 +12,28 @@
       # Ideally we would source tools/installPythonClient.R to not
       # have to duplicate the synapseclient install code
       # system2(paste("Rscript ", getwd(), "/tools/installPythonClient.R ", getwd(), sep=""))
-      PYTHON_CLIENT_VERSION <- '4.0.0'
+      PYTHON_CLIENT_VERSION <- '4.4.0'
       # reticulate::virtualenv_create('r-reticulate')
       # reticulate::use_virtualenv('r-reticulate')
-      reticulate::py_install(c("requests<3", "pandas~=2.0.0", "pysftp", "jinja2", "markupsafe"))
+      reticulate::py_install(c("requests<3", "pandas==2.0.3", "pysftp", "jinja2", "markupsafe","numpy==1.24.4"))
```

Comment:
Should these be ranges that dictates the max version we would allow?

`pandas<=2.0.3`
`numpy<=1.24.4`

Reply:
I can remove requests and pysftp installation since I checked installing synapseclient does install requests < 3 and pysftp-0.2.9 as what listed in the pipfile of the synapseclient. However, I'm a bit concern about installing pandas since it installs pandas-2.2.2 and does not align with pipfile of the synapseclient. I locked down the package version to avoid dependency errors. But I'm happy to test if we can go with version range for pandas and numpy. Will keep you guys updated.

Reply:
I removed requests and pysftp and use version range for pandas and numpy. The build run through successfully.